src/styles/StyleUtils.js

```diff
@@ -874,42 +874,52 @@ function getEmojiSuggestionContainerStyle(
  */
 const getColoredBackgroundStyle = isColored => ({backgroundColor: isColored ? colors.blueLink : null});
 
-function getEmojiReactionBubbleStyle(isHovered, hasUserReacted, sizeScale = 1) {
-    const sizeStyles = {
-        paddingVertical: styles.emojiReactionBubble.paddingVertical * sizeScale,
-        paddingHorizontal: styles.emojiReactionBubble.paddingHorizontal * sizeScale,
-    };
+function getEmojiReactionBubbleStyle(isHovered, hasUserReacted, isContextMenu = false) {
+    let backgroundColor = themeColors.border;
 
     if (hasUserReacted) {
-        return {backgroundColor: themeColors.reactionActive, ...sizeStyles};
+        backgroundColor = themeColors.reactionActive;
     }
+
     if (isHovered) {
-        return {backgroundColor: themeColors.buttonHoveredBG, ...sizeStyles};
+        backgroundColor = themeColors.buttonHoveredBG;
     }
 
-    return sizeStyles;
-}
+    if (isContextMenu) {
+        return {
+            paddingVertical: 3,
+            paddingHorizontal: 12,
+            backgroundColor,
+        };
+    }
 
-function getEmojiReactionTextStyle(sizeScale = 1) {
     return {
-        fontSize: styles.emojiReactionText.fontSize * sizeScale,
-        lineHeight: styles.emojiReactionText.lineHeight * sizeScale,
+        paddingVertical: 2,
+        paddingHorizontal: 8,
+        backgroundColor,
     };
 }
 
-function getEmojiReactionCounterTextStyle(hasUserReacted, sizeScale = 1) {
-    const sizeStyles = {
-        fontSize: styles.reactionCounterText.fontSize * sizeScale,
+function getEmojiReactionBubbleTextStyle(isContextMenu = false) {
+    if (isContextMenu) {
+        return {
+            fontSize: 17,
+            lineHeight: 28,
+        };
+    }
+
+    return {
+        fontSize: 15,
+        lineHeight: 24,
     };
+}
 
+function getEmojiReactionCounterTextStyle(hasUserReacted) {
     if (hasUserReacted) {
-        return {
-            ...sizeStyles,
-            color: themeColors.link,
-        };
+        return {color: themeColors.textLight};
```

Comment:
I don't see any proposal or approval regarding this change in color. Can you please add a video to get approval from the design team.

Reply:
@daraksha-dk There is also change in `backgroundColor` of EmojiReactionBubble on hover which is not present in staging.

https://user-images.githubusercontent.com/25876548/230912707-f5a335a6-e26e-4ce0-9559-5c63b6e4233a.mov